lib/evaluate_board.rb

```diff
@@ -1,3 +1,5 @@
+# frozen_string_literal: true
+
 class EvaluateBoard
```

Comment:
Thinking `@winning_combinations` should be a constant instead of an instance variable?

Reply:
I think so too! Will fix this.